Source/vtkPyFR/vtkPyFRAdaptor.h

```diff
@@ -10,6 +10,13 @@ extern "C"
   void CatalystFinalize(void* p);
 
   void CatalystCoProcess(double time, unsigned int timeStep, void* p, bool lastTimeStep=false);
+
+  void CatalystCamera(void* p, const float eye[3], const float ref[3],
+                      const float vup[3]);
+
+  void CatalystBGColor(void* p, const float color[3]);
+
+  void CatalystImageResolution(void* p, const float color[3]);
```

Comment:
Typo: the `CatalystImageResolution` argument should be a `const uint32_t [2]`as opposed to a `const float [3]`.